nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy

```diff
@@ -81,6 +94,7 @@ class ConanProxyRecipe
 
     builder.route(new Route.Builder()
         .matcher(BrowseUnsupportedHandler.MATCHER)
+        .handler(highAvailabilitySupportHandler)
```

Comment:
Is HA handler after SecurityHandler in the chain.

Reply:
After additional testing moved handler into v1 Controller. Not it is explicit after Security handler 72f00903464847a5a37566837c1bfffe22804906